### DISCUSSION TITLE
Confused by the documentation

### DISCUSSION THREAD

**Discussion by cwhy** [nrdxp/nixdev#361] [Q&A] [answered]
Hi I am new to NixOS and want to try a friendly setup. I am quite confused at this page, which is a essential page for setup the system: `https://devos.divnix.com/start/iso.html`. The page requires me to install `bud` but I see no documentation for how to install `bud` from `https://devos.divnix.com/bud/index.html` or the repo.

Any help?
Thanks~

**Comment by Pacman99** [comment]
You don't exactly need to "install" bud, it is included in the devshell of the latest devos. So you should be able to do `nix develop` and access the `bud` command.

**Comment by astratagem** [comment, marked as answer]
I also ran into the same initial confusion after following the instructions on the docs. The "Quick Start" page and some of the other pages seem to be out of sync with one another. 

"Quick Start" has instructions for installing DevOS with the following command:

```sh
nix-shell https://github.com/divnix/devos/archive/core.tar.gz -A shell \
  --run "flk get core"
```

That will work – it gives you the `core` branch. But on the `core` branch, only `flk` is available, not `bud`. I'm guessing that's why @cwhy was under the impression that they needed to install it (but please correct me if I'm assuming too much). So was I, until I realized `bud` was added in a commit later than the last commit on `core`.

My solution to that was to ignore the "Quick Start" docs and do something like the following:

```sh
git clone git@github.com:divnix/devos.git
cd devos
rm -rf .git
nix develop
```

My two cents: it seems like the Quick Start docs should either be updated to point to `master` and use `bud` instead of `flk`, or the references to `bud` in the next few pages of the docs should be reverted to reflect the state of DevOS on the `core` branch. I think one of these would help clarify things for new users. But `bud` looks pretty nice, and I get that it replaces `flk`, so maybe updating Quick Start would be more future-friendly.

**Reply by astratagem** [reply]
Opened #366

**Reply by blaggacao** [reply]
Hm, it's a bit of a mess. I guess we're keeping the core branch to not have existing old pins on core stop working.

There should be no mention at all in the docs about it.

**Comment by DrPyser** [comment]
I also found the documentation confusing. The Quick Start relies on the 'bud' command without introducing it. 

Also, current version of the repository seems not to provide `bud`?

```
nix-shell https://github.com/divnix/devos/archive/main.tar.gz -A shell  --run "bud get main"
/tmp/nix-shell-782154-0/rc: line 3: bud: command not found
```
